enum/Analytics.ts

```diff
@@ -46,6 +47,14 @@ export const Events: Record<string, IAnalyticsEvent> = {
 			failure_reason: 'network/app error in SF app',
 		},
 	},
+	[EventName.ESCALATION_FAILED_DUE_TO_DISCONNECTED_AGENT]: {
+		category: EventCatagory.ESCALATION,
+		eventType: EventType.SESSION,
+		action: 'failed',
```

Comment:
I wonder if this should be a unique action, since it's not really a failed escalation.  It's more like "interrupted" maybe.  In order to do that however we'll need the Pixel team to sign off on a new action.  I can work on that.